0.download_data/scripts/1.view_the data_structure.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook looks at the downloaded mnist data and looks into the image dataloader and rolling window data class.
+# Noteably this code is modified from the original code written by original code by Xinqiang Ding <xqding@umich.edu>
```

Comment:
```suggestion
# Noteably, this code is modified from the original code written by Xinqiang Ding <xqding@umich.edu>
```
Might recommend a direct link to the code you are using in this file, if applicable.

Reply:
Goood point, I added the code linke too